build-ps/percona-server.spec

```diff
@@ -1341,6 +1344,9 @@ fi
 %dir %attr(755, mysqlrouter, mysqlrouter) /var/run/mysqlrouter
 
 %changelog
+* Fri Feb 12 2021 Illia Pshonkin <illia.pshonkin@percona.com> - 8.0.22-13
```

Comment:
As we are doing this automatically let's use:
Percona Development Team <info@percona.com>